src/main/java/com/google/step/snippet/external/GeeksForGeeksClient.java

```diff
@@ -11,6 +11,7 @@
   private static final String DESC_TAG = "p";
   private static final String SNIPPET_CLASS = "code-block";
   private static final String CODE_CLASS = "code-container";
+  private static final String SOURCE_NAME = "GeeksForGeeks";
```

Comment:
I don't think we should implicitly tie these strings with the names of files. If you want this to be the name of a file, be explicit and say "geeksforgeeks.png".

However, I don't think we can add these images into the git repo. We don't own them, and we don't have the license to use them like that. Instead, can we link to the icon itself? For example, GeeksForGeeks' is at https://www.geeksforgeeks.org/favicon.ico